src/ProfilesPage/ProfilePage.js

```diff
@@ -75,8 +84,10 @@ class ProfilePage extends Component {
   }
 
   renderEditUserButton() {
+    console.log(this.state.editUserEnable);
     return (
       <button className="btn btn-info" onClick={this.handleEditButtonClick.bind(this)}>
+        {/*if this.state.editUserEnable === false render "back" else render "Edit"*/}
```

Comment:
Can just remove the commented stuff.

Reply:
this seems more of a note explaining what the functioning is doing